src/main/java/http/request/FirstLine.java

```diff
@@ -0,0 +1,55 @@
+package http.request;
+
+import http.EnumHttpMethod;
+import java.io.UnsupportedEncodingException;
+import java.net.URLDecoder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class FirstLine {
+
+  private static final Logger logger = LoggerFactory.getLogger(FirstLine.class);
+  EnumHttpMethod method;
+  String url;
+  String version;
+
+  public FirstLine(String method, String url, String version) {
+    this(EnumHttpMethod.valueOf(method), url, version);
+  }
+
+  public FirstLine(EnumHttpMethod method, String url, String version) {
+    setMethod(method);
+    setUrl(url);
+    setVersion(version);
+  }
+
+  public void setMethod(EnumHttpMethod method) {
+    this.method = method;
+  }
+
+  public void setUrl(String url) {
+    try {
+      this.url = URLDecoder.decode(url, "UTF-8");
```

Comment:
이미 선언되어 있는 상수를 활용할 수 있네요. 재사용 가능한 상수를 활용할 수 있다면 오타도 줄이고, 추후 언젠가 발생할 변경사항으로 인한 상위호환/하위호환을 보장할 수 있는 등 여러 장점이 있습니다
```suggestion
      this.url = URLDecoder.decode(url, StandardCharsets.UTF_8);
```

Reply:
https://github.com/kakao-2022-summer/be-w234-was/pull/55/commits/b573b8f789881629fb10bda7fe5d97b4b7270a21

반영했습니다